driver-core/src/main/com/mongodb/UnixServerAddress.java

```diff
@@ -17,12 +17,14 @@
 package com.mongodb;
 
 import com.mongodb.annotations.Immutable;
+import com.mongodb.internal.graalvm.substitution.SocketStreamFactorySubstitution;
 
 import static com.mongodb.assertions.Assertions.isTrueArgument;
 import static com.mongodb.assertions.Assertions.notNull;
 
 /**
  * Represents the location of a MongoD unix domain socket.
+ * It is {@linkplain SocketStreamFactorySubstitution#create(ServerAddress) not supported in GraalVM native image}.
```

Comment:
Note that because internal classes are excluded from our API docs, this `linkplain` produces plain text "not supported in GraalVM native image" in the generated API docs. But in an IDE we still have a working link, which is convenient. The same goes for the change in `MongoCompressor`.